data-model/src/main/java/org/alfresco/util/ConfigScheduler.java

```diff
@@ -57,7 +57,7 @@
     public static class ConfigSchedulerJob implements Job
     {
         @Override
-        public void execute(JobExecutionContext context) throws JobExecutionException
+        public synchronized void execute(JobExecutionContext context) throws JobExecutionException
```

Comment:
Has no effect in normal operation. On a laptop which is put to sleep, there can be a number of Threads calling execute is quick succession. Make debug a little difficult.